content/articles/javascript-canvas-piechart/index.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /javascript-canvas-piechart/
+title: Creating a pie chart from scratch using JavaScript, HTML Canvas, and CSS
+description: This article will discuss how to create a pie chart from scratch using JavaScript, HTML Canvas, and CSS.
+author: sandra-moringa
+date: 2021-12-16T00:00:00-21:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+ - url: /engineering-education/javascript-canvas-piechart/hero.jpg
+   alt: JavaScript Canvas Piechart Example Image
+---
+In some cases, you may want to create a chart without any library. That means you will have to create one from scratch. We will see how to do that using JavaScript, HTML Canvas, and CSS.
+<!--more-->
+
+### Prerequisites
+To follow through this, you will be required to have a basic understanding of these:
+1. HTML
+2. CSS
+3. JavaScript
+
+### An overview of the canvas
+It is easy to position, give custom shapes and color to items in the DOM using the CSS styling provided. Some operations, however, like drawing lines between two or more locations, are particularly difficult to accomplish with standard HTML components. That being said, there are two other options provided:
+1. SVG(Scalable Vector Graphics)
+2. Canvas
+
+ SVG is used to create vector graphics, as the name suggests, in XML format. This article focuses on the canvas, so we won't go further into SVG. Read more about it [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg) if you are interested in SVG. 
```

Comment:
It's much more grammatically correct when you start with, "As the name suggests" rather than placing it inside the sentence.